src/main/java/carracing/domain/service/CarRacingServiceImpl.java

```diff
@@ -0,0 +1,66 @@
+package carracing.domain.service;
+
+import carracing.domain.dto.RacingData;
+import carracing.domain.dto.RacingResult;
+import carracing.domain.dto.RoundData;
+import carracing.domain.dto.RoundResult;
+import carracing.domain.entity.Car;
+import carracing.domain.entity.Cars;
+import carracing.domain.entity.Participant;
+import carracing.domain.entity.Round;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static java.util.stream.Collectors.collectingAndThen;
+import static java.util.stream.Collectors.toList;
+
+public class CarRacingServiceImpl implements CarRacingService{
+
+  private static final String GAME_START_ERROR_MESSAGE = "게임을 시작할 수 없습니다.";
+
+  private MoveStrategy moveStrategy;
+
+  public CarRacingServiceImpl(MoveStrategy moveStrategy) {
+    this.moveStrategy = moveStrategy;
+  }
+
+  @Override
+  public Cars registerRacer(Participant participant) {
+    Cars cars = new Cars();
+    for (int id = 0; id < participant.size(); id++) {
+      cars.register(new Car(participant.getParticipant().get(id)));
+    }
```

Comment:
```suggestion
    for(Name name : participant) {
        cars.register(new Car(name));
    }
```

이렇게도 가능할 것 같아요. 나중에 시간이 되시면 한 번 해보시는것도 도움이 될 거에요 ㅎ. 
(추가적인 방법 제안이지 이 코드가 문제있다는건 아닙니다. )